src/hooks/useIpc.js

```diff
@@ -48,7 +53,7 @@ const useIpc = (history) => {
 
     ipc.on('update:downloading', (action, { label }) => {
       toast.success(label);
-    });
+    }); */
```

Comment:
This change will be reverted after approved. They were meant to bypass ipc and directly show the message